platform/ui/src/components/Thumbnail/Thumbnail.jsx

```diff
@@ -35,7 +35,7 @@ const Thumbnail = ({
 
   return (
     <div
-      onFocus={() => blurHandlerListener(thumbnailElement)}
+      onFocus={() => clickOutsideHandler(thumbnailElement)}
```

Comment:
Ah, this is why. Yeah. Just duplicate the method, and give it a different name.
I'd rather it be explicit what the method does.